Sources/Moya/MoyaProvider+Defaults.swift

```diff
@@ -30,4 +29,12 @@ public extension MoyaProvider {
         manager.startRequestsImmediately = false
         return manager
     }
+
+    private final class func url(for target: Target) -> URL {
```

Comment:
Let's add a comment here with the link to the issue and a rationale for why this is being done